bat-utils/lib/runtime-currency.js

```diff
@@ -340,7 +340,6 @@ const maintenance = async (config, runtime) => {
   if (flatlineP) {
     debug('maintenance', { message: 'no trades reported' })
     runtime.captureException(new Error('maintenance reports flatline'))
-    process.exit(0)
```

Comment:
we were only hitting this branch on select services that were misconfigured to not use the helper service to retrieve exchange rate information. should we still drop this?

how do you feel about failing out if helper is not configured when env = prod?

Reply:
the `process.exit` is called only when helper is not configured (look at the top of the function!) . i've made the other change you suggest...